src/map/lua/lua_baseentity.cpp

```diff
@@ -10766,6 +10785,62 @@ int32 CLuaBaseEntity::getAutomatonHead(lua_State* L)
     return 1;
 }
 
+int32 CLuaBaseEntity::isSpawned(lua_State* L)
+{
+    DSP_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+    return ((CMobEntity*)m_PBaseEntity)->PAI->IsSpawned();
```

Comment:
`lua_pushTYPE(L, var);` where `TYPE` is `boolean`, `integer` etc (see how other functions return)
and then `return 1;`